vcxagency-node/src/configuration/app-config.js

```diff
@@ -67,49 +67,52 @@ const configValidation = Joi.object().keys({
   PG_WALLET_ADMIN_PASSWORD_SECRET: Joi.string(),
 
   PG_WALLET_URL: Joi.string().required(),
-  PG_WALLET_MAX_CONNECTIONS: Joi.number().integer().min(50).max(999),
-  PG_WALLET_MIN_IDLE_COUNT: Joi.number().integer().min(0).max(0), // max 1 enforced on plugin level for MultiWalletSingleTableSharedPool strategy
-  PG_WALLET_CONNECTION_TIMEOUT_MINS: Joi.number().integer().min(1).max(100),
+  PG_WALLET_MAX_CONNECTIONS: Joi.number().integer().min(50).max(999).default(90),
+  PG_WALLET_MIN_IDLE_COUNT: Joi.number().integer().min(0).max(0).default(0), // max 1 enforced on plugin level for MultiWalletSingleTableSharedPool strategy
```

Comment:
This is actually a convoluted declaration of a constant :) But I guess this is in preparation of the future where the maximum minimum is increased?

Reply:
removed